crates/wasi-provider/src/states/completed.rs

```diff
@@ -0,0 +1,16 @@
+use crate::PodState;
+use kubelet::state::prelude::*;
+
+state!(
+    /// Pod was deleted.
```

Comment:
Wait.... wouldn't a Deleting state be different from a Completed state?  E.g. a job pod could complete but would remain in existence.

Reply:
okay. Renamed to `Terminated`.